CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Add model and API endpoints to manage a shared live media
 - Add component to display chat, viewers and apps during a live in a panel
+- Add `consumer_site` in JWT token
```

Comment:
In your commit message: 

- `s/The consumer site is sent during the LTI connexion./The consumer site is sent during the LTI connection./`
- `s/Models in Marsha due to LTI’s protocol and video’s portability are sometimes complex./Models in Marsha due to LTI’s protocol and resources' portability are sometimes complex./`
- `s/a playlist with a context_id can have multiple consumer_site./a playlist for a given context_id can have multiple consumer_sites./`
- `s/It makes it hard to link a user with his consumer site./It makes it hard to link a resource's playlist for a given context_id with its consumer site./`